src/windows/MediaProxy.js

```diff
@@ -226,6 +301,10 @@ module.exports = {
         try {
             if (thisM.node) {
                 thisM.node.onloadedmetadata = null;
+                // Unsubscribing as the media object is being released
+                thisM.node.onerror = null;
+                // Needed to avoid "0x80070005 - JavaScript runtime error: Access is denied." on copyAsync
```

Comment:
This occurs when there is an Audio object with `src` set to a file being rewritten on recording.